src/couch_replicator_scheduler.erl

```diff
@@ -298,7 +312,11 @@ backoff_micros(CrashCount) ->
 
 -spec add_job_int(#job{}) -> boolean().
 add_job_int(#job{} = Job) ->
-    ets:insert_new(?MODULE, Job).
+    InsertRes = ets:insert_new(?MODULE, Job),
+    TotalJobs = ets:info(?MODULE, size),
+    couch_stats:update_gauge([couch_replicator, jobs, total], TotalJobs),
+    couch_stats:increment_counter([couch_replicator, jobs, adds]),
```

Comment:
this is tracking the number of calls to add_job_int or the number of times we add a job? If the latter, it is wrong if InsertRes is false.